gremlin-go/driver/driverRemoteConnection.go

```diff
@@ -47,16 +47,17 @@ type DriverRemoteConnectionSettings struct {
 	NewConnectionThreshold int
 	// Maximum number of concurrent connections. Default: number of runtime processors
 	MaximumConcurrentConnections int
+	session                      string
```

Comment:
This should probably be either at the top of the list or at the bottom and separated out to make clear that it's an internal field.